Adding some Drupal and Wordpress love to `terminus art`

ElliotFriend opened the pull request:
I mean, pantheon and unicorns are awesome, but I just wasn't feeling the love for drupal and wp :smiley: 

Signed-off-by: Elliot Voris elliot@voris.me

ElliotFriend committed:
Adding some Drupal and Wordpress love to `terminus art`

Signed-off-by: Elliot Voris <elliot@voris.me>

ElliotFriend committed:
Making `terminus art` return a random piece of art

I love seeing the Pantheon logo when I login, but some variety
might be nice every now and then :)

Signed-off-by: Elliot Voris <elliot@voris.me>

joshkoenig commented:
+1

We should make "adding art" one of the suggested first/easiest contributions. Also possibly a great way to put an April Fools surprise in place... ;)

mikevanwinkle commented:
Hmmm ... should we have the login art on random @joshkoenig ... right now everyone gets the terminus brand art but this would mean only 1/4 would say terminus on it. I'm cool with having the art available just questioning the randomizing after login. Thoughts?

ElliotFriend commented:
@mikevanwinkle I had that thought, too. And, the longer I thought of it, the more it makes sense to me.

Maybe a better place for randomization would after somebody issues `terminus art`?

ElliotFriend committed:
Merge branch 'master' of github.com:pantheon-systems/cli into terminus-art

ElliotFriend committed:
PR #123: Removing the art randomization after a successful login

It doesn't make any sense to randomize the artwork after a login
has been successfully completed, for a bunch of reasons.

Now, the Terminus fist will always be displayed after a login.
`terminus art` will result in a randomly chosen piece.
`terminus art <piece>` will display that particular piece.

Signed-off-by: Elliot Voris <elliot@voris.me>
